### DISCUSSION TITLE
front end and back end connection

### DISCUSSION THREAD

**Discussion by yolobolo69** [community/community#142555] [Programming Help]
### Body

i have an app  built by php how i connect it whith xaamp 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by mostaphabr** [comment]
easy , import you project directory to the htdocs folder in the xamp folder

**Reply by yolobolo69** [reply]
goood

**Comment by ebndev** [comment]
Hi @yolobolo69 & @mostaphabr, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. 

Thanks for understanding and we hope to see you participate in other discussions in the future!
